backend/index.js

```diff
@@ -17,13 +17,22 @@ axios
   })
 
 app.use(express.json())
+
 app.get('/cards', (req, res) => {
   res.json(db.getCards())
 })
+
 app.get('/classCards/:className', (req, res) => {
   res.json(db.getCardsByClass(req.query.className))
 })
 
+app.post('/addDeck', (req, res, next) => {
+  const deck = {}
+  deck[req.query.name] = req.body
```

Comment:
Consider destructuring `req.body` to make it clear what the deck object properties are.

Reply:
req.body will be an array of cards written as Jsons. How i would destruct that?

Reply:
Disregard my initial suggestion.

You shouldn't need an array of objects in the req.body. The server database already holds the card data. Wouldn't an array of card ids suffice?
```js
[id1, id3, id175]
```
Consider commenting your code. Also, this format would indicate that you are expecting an array:
```js
const cardIds = req.body
deck[deckName] = cardIds
```
You still need to make sure that every id received is valid before committing to database.

Reply:
@ionhoria  Check now